shift/rename_beesdoo.py

```diff
@@ -11,23 +11,138 @@
+        ("beesdoo_shift_shift_id", "shift_shift_id"),
+    ],
+    "res_partner_shift_template_rel": [
+        ("beesdoo_shift_template_id", "shift_template_id")
+    ],
+    "cooperative_status_shift_journal_rel": [
+        ("beesdoo_shift_journal_id", "shift_journal_id")
+    ],
+}
+CONSTRAINTS_TO_RENAME = {
+    "cooperative_status_shift_journal_rel": [
+        (
+            "beesdoo_shift_journal_coopera_beesdoo_shift_journal_id_coop_key",
+            "cooperative_status_shift_jour_shift_journal_id_cooperative__key",
+        ),
+        (
+            "beesdoo_shift_journal_cooperative_beesdoo_shift_journal_id_fkey",
+            "cooperative_status_shift_journal_rel_shift_journal_id_fkey",
+        ),
+        (
+            "beesdoo_shift_journal_cooperative_st_cooperative_status_id_fkey",
+            "cooperative_status_shift_journal_rel_cooperative_status_id_fkey",
+        ),
+    ],
+    "res_partner_shift_template_rel": [
+        (
+            "beesdoo_shift_template_res_pa_beesdoo_shift_template_id_res_key",
+            "res_partner_shift_template_re_shift_template_id_res_partner_key",
+        ),
+        (
+            "beesdoo_shift_template_res_partn_beesdoo_shift_template_id_fkey",
+            "res_partner_shift_template_rel_shift_template_id_fkey",
+        ),
+    ],
+    "shift_assign_super_coop_shift_shift_rel": [
+        (
+            "beesddoo_shift_assign_super_c_beesddoo_shift_assign_super_c_key",
+            "shift_assign_super_coop_shift_shift_assign_super_coop_id_sh_key",
+        ),
+        (
+            "beesddoo_shift_assign_super_c_beesddoo_shift_assign_super__fkey",
+            "shift_assign_super_coop_shift_s_shift_assign_super_coop_id_fkey",
+        ),
+        (
+            "beesddoo_shift_assign_super_coop_be_beesdoo_shift_shift_id_fkey",
+            "shift_assign_super_coop_shift_shift_rel_shift_shift_id_fkey",
+        ),
+    ],
+    "shift_temporary_exemption": [
+        (
+            "beesdoo_shift_temporary_exempti_temporary_exempt_reason_id_fkey",
+            "shift_temporary_exemption_temporary_exempt_reason_id_fkey",
+        ),
+    ],
 }
+INDEXES_TO_RENAME = [
+    (
+        "beesddoo_shift_assign_super_c_beesddoo_shift_assign_super_c_idx",
+        "shift_assign_super_coop_shift_sh_shift_assign_super_coop_id_idx",
+    ),
+    (
+        "beesddoo_shift_assign_super_coop_bee_beesdoo_shift_shift_id_idx",
+        "shift_assign_super_coop_shift_shift_rel_shift_shift_id_idx",
+    ),
+    (
+        "beesdoo_shift_journal_cooperative__beesdoo_shift_journal_id_idx",
+        "cooperative_status_shift_journal_rel_shift_journal_id_idx",
+    ),
+    (
+        "beesdoo_shift_journal_cooperative_sta_cooperative_status_id_idx",
+        "cooperative_status_shift_journal_rel_cooperative_status_id_idx",
+    ),
+    (
+        "beesdoo_shift_template_res_partne_beesdoo_shift_template_id_idx",
+        "res_partner_shift_template_rel_shift_template_id_idx",
+    ),
+]
```

Comment:
The naming schemes appear very different here. How is this possible?

Also, are you sure this doesn't contain typos? There's a `res_partne`, a double `__`, a `cooperative_sta`, etc etc etc.